lib/streamlit/runtime/connection_factory.py

```diff
@@ -39,6 +39,8 @@
 }
 MODULE_EXTRACTION_REGEX = re.compile(r"No module named \'(.+)\'")
 MODULES_TO_PYPI_PACKAGES: Final[Dict[str, str]] = {
+    "MySQLdb": "mysql-connector-python",
```

Comment:
This should actually be

```
"MySQLdb": "mysqlclient",
```

(Just tested, and Zachary reported the same thing)

Reply:
Whoops, must have missed the correction in the notion doc. Fixed